Sources/SwiftParser/Declarations.swift

```diff
@@ -476,6 +476,10 @@ extension Parser {
             arena: self.arena
           )
         )
+        if self.at(prefix: ">") {
+          break
+        }
+            
```

Comment:
Why is this needed? Could you add a comment for it?

Reply:
Actually I was trying trailing comma for generics and forget to remove this.